bin/seviri_level1c_runner.py

```diff
@@ -21,42 +21,39 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
-from seviri_l1c_runner import get_config
-
-from level1c4pps.seviri_hrit import process_one_scan
+from level1c4pps.seviri2pps_lib import process_one_scan  # @UnresolvedImport
+from posttroll.subscriber import Subscribe  # @UnresolvedImport
 from posttroll.publisher import Publish  # @UnresolvedImport
 from posttroll.message import Message  # @UnresolvedImport
 
 import logging
 import sys
 import os
-import six
-from posttroll.subscriber import Subscribe  # @UnresolvedImport
 from multiprocessing import cpu_count
+
+import six
 if six.PY2:
-    from urlparse import urlunsplit
+    from urlparse import urlunsplit  # @UnusedImport
 elif six.PY3:
-    from urllib.parse import urlunsplit  # @UnresolvedImport
+    from urllib.parse import urlunsplit  # @UnresolvedImport @Reimport
 import socket
 
-from seviri_l1c_runner.utils import (deliver_output_file, cleanup_workdir)
+from seviri_l1c_runner.utils import (deliver_output_file, cleanup_workdir, get_config)  # @UnresolvedImport
 
 SUPPORTED_METEOSAT_SATELLITES = ['meteosat-8', 'meteosat-9', 'meteosat-10', 'meteosat-11']
 
-
 #: Default time format
 _DEFAULT_TIME_FORMAT = '%Y-%m-%d %H:%M:%S'
 
 #: Default log format
 _DEFAULT_LOG_FORMAT = '[%(levelname)s: %(asctime)s : %(name)s] %(message)s'
 
+#: SMHI MODE
+MODE = os.environ.get('SMHI_MODE', 'utv')
```

Comment:
To make this not SMHI-centric, could this be passed as a command line argument instead ? For example `-e` for environment ?

Reply:
Yes, the SMHI_MODE is something that is supposed to disappear. I just haven't taken the time to do it jet.